rpcs3/cmake_modules/ConfigureCompiler.cmake

```diff
@@ -6,6 +6,16 @@ if(MSVC)
 	set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /NODEFAULTLIB:libc.lib /NODEFAULTLIB:libcmt.lib /NODEFAULTLIB:libcd.lib /NODEFAULTLIB:libcmtd.lib /NODEFAULTLIB:msvcrtd.lib")
 	set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /SUBSYSTEM:WINDOWS /DYNAMICBASE:NO /BASE:0x10000 /FIXED")
 
+	#TODO: Some of these could be cleaned up
+	add_compile_options(/wd4244) # Implictly downcasting from larger integer type or converting float to int
+	add_compile_options(/wd4309) # Truncation of constant value
```

Comment:
Those two are usefull warnings.
Without warning about trunction you wouldnt find CELL_HTTPS_ERROR_RETRY_CONNECTION typo.